include/dxc/dxctools.h

```diff
@@ -29,19 +29,22 @@ struct IDxcRewriter : public IUnknown {
                                                         _In_z_ LPCWSTR entryPoint,
                                                         _In_count_(defineCount) DxcDefine *pDefines,
                                                         _In_ UINT32 defineCount,
+                                                        _In_count_(argCount) LPCSTR *pArgs, _In_ UINT32 argCount,
```

Comment:
Can't just change existing interface.  Need to use the new IDxcRewriter2::RewriteWithOptions instead, which supports passing options.